src/Loader/FilesystemLoader.php

```diff
@@ -170,7 +170,7 @@ public function exists($name)
         }
 
         try {
-            return false !== $this->findTemplate($name, false);
+            return false !== $this->findTemplate($name);
```

Comment:
this change looks wrong to me. We have this as a hidden argument.

Reply:
indeed :)